src/gc/impl/conservative/gc.d

```diff
@@ -2557,11 +2580,17 @@ struct Gcx
         return recoverPool[bin] = poolIndex < npools ? cast(SmallObjectPool*)pool : null;
     }
 
+    void disableFork() nothrow
+    {
+        config.fork = false;
```

Comment:
Maybe better to not modify the configuration, but remember the failure in a member variable. Printing a warning might also be useful.

Reply:
The GC can use a bool to remember if it should fork or not.
In any case it would be useful to give the user an option to disable the forking behaviour in certain section of user code.
If you think that is needed, I will add a enableFork function

Reply:
There is currently no way to add a custom member function to the GC interface, so it would be a bit problematic for the user to call it.
Instead core.gc.config can be imported and config.fork can be modified, so the GC can adapt accordingly, but it should not modify the user setting.
I think you should just remember whether forking failed in a previous attempt and then ignore `config.fork`.